com.unity.multiplayer.mlapi/Editor/MLAPIProfilerModule.cs

```diff
@@ -68,7 +68,8 @@ private static List<MLAPIProfilerCounter> CreateMessagesCounters() => new List<M
             new MLAPIProfilerCounter { m_Name = ProfilerConstants.UnnamedMessageReceived, m_Category = ProfilerCategory.Network.Name },
             new MLAPIProfilerCounter { m_Name = ProfilerConstants.ByteSent, m_Category = ProfilerCategory.Network.Name },
             new MLAPIProfilerCounter { m_Name = ProfilerConstants.ByteReceived, m_Category = ProfilerCategory.Network.Name },
-            new MLAPIProfilerCounter { m_Name = ProfilerConstants.NetworkVarReceived, m_Category = ProfilerCategory.Network.Name },
+            new MLAPIProfilerCounter { m_Name = ProfilerConstants.NetworkVarUpdates, m_Category = ProfilerCategory.Network.Name },
+            new MLAPIProfilerCounter { m_Name = ProfilerConstants.NetworkVarDeltas, m_Category = ProfilerCategory.Network.Name },
```

Comment:
Can you help me understand the difference between these two stats?

Reply:
One does uses buffer manager and one is the normal expected network var changes.
However the best person to answer this would be Luke or Albin

I was just clarifying the name as these were two distinct things